File: Interview Preparation Kit/Recursion and Backtracking/Recursion Fibonacci Numbers/Fabonacci.java
import java.util.Scanner;

public class Fabonacci {
	static int Recursion(int x) {
		if (x < 2) {
			return x > 0 ? 1 : 0;
		} else {
			return Recursion(x - 1) + Recursion(x - 2);
		}
	}

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		int n = input.nextInt();
		System.out.println(Recursion(n));
	}
}

File: Interview Preparation Kit/Recursion and Backtracking/Recursion Fibonacci Numbers/fabonacci.cpp
#include <iostream>
using namespace std;

int Fabonacci(int x) {
	if (x < 2) {
		return x > 0 ? 1 : 0;
	} else {
		return Fabonacci(x - 1) + Fabonacci(x - 2);
	}
}

int main()
{
	int n = 0;
	cin >> n;

	cout << Fabonacci(n);
}